### ISSUE TITLE
Hidden selection indicator doesn't turn off 

### ISSUE THREAD

**Issue by EdAyers** [71/dance#185]
Hi the solution in #184 did not work for me. I'm on Dance 0.5.2.
If I set `"hiddenSelectionsIndicatorsDecoration": {},` it does not prevent the decorations from being pushed to the editor. For me it still adds the text but with no decoration (see top line of image).
The solution I guess is to give an explicit `decorationType === "off"` (or similar) condition to https://github.com/71/dance/blob/86afe9bd0a50f81d310f65cc113818e9137e3f65/src/state/editors.ts#L403

![image](https://user-images.githubusercontent.com/5064353/126234873-d4075298-2afa-4827-bbca-ae8b706e9e6c.png)

Thanks for making this extension!

**Comment by 71** [comment]
Hey there,

Actually, the fix mentioned in #184 addresses specifically this issue, but hasn't been submitted to the marketplace yet. In the meantime you can set the color to `editor.background`, which essentially makes the decoration invisible.

```json
"hiddenSelectionsIndicatorsDecoration": {
    "after": {
        "color": "$editor.background"
    },
    "isWholeLine": true
}
```

**Comment by EdAyers** [comment]
Thanks!
